starboard/tools/testing/test_report_parser.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Parses the gtest xml output and prints the failing tests"""
+
+import pathlib
+import sys
+
+from junitparser import JUnitXml
+
+if __name__ == '__main__':
+  if len(sys.argv) != 2:
+    print(f'Usage: {sys.argv[0]} <path to unit test reports>')
+    sys.exit(1)
+
+  print('======== UNIT TEST REPORT ========')
+  success = True
+  for report in pathlib.Path(sys.argv[1]).rglob('*.xml'):
+    # Get the target name from the filename generated by test_runner.py.
+    target_name = pathlib.Path(report).stem
+    try:
+      xml = JUnitXml.fromfile(report)
+      failure_str = ''
+      for suite in xml:
+        for case in suite:
+          if not case.is_passed:
```

Comment:
Is it an invariant that `if not case.is_passed` then `len(case.result) > 0`?

Reply:
More explicitly I was wondering why we set `success=False` in the for loop instead of directly after `if not case.is_passed` but it's since been changed so lgtm